app/survey_config/survey_config.py

```diff
@@ -74,10 +71,14 @@ def get_footer_legal_links(  # pylint: disable=unused-argument, no-self-use
     ) -> Optional[list[dict]]:
         return None
 
-    def get_data_layer(  # pylint: disable=no-self-use
-        self, tx_id: Optional[str] = None
-    ) -> list[dict]:
-        if tx_id:
-            return [{"tx_id": tx_id}]
-
-        return []
+    def get_data_layer(self, tx_id: Optional[str] = None) -> list[dict]:
```

Comment:
The overridden `get_data_layer` in the other classes need to be removed, since you have updated the base class.

Reply:
Ive removed the business one but left the census one